_i18n/de/resources/moneropedia/ringsignatures.md

```diff
@@ -1,17 +1,17 @@
 ---
-terms: ["ring-signature", "ring-signatures"]
-summary: "a group of cryptographic signatures with at least one real participant, but no way to tell which in the group is the real one as they all appear valid"
+terms: ["ring-signature", "ring-signatures", "Ringsignatur", "Ringsignaturen"]
+summary: "Eine Gruppe kryptografischer Signaturen mit mindestens einem realen Beteiligten, aber keiner Möglichkeit, auszumachen, wer in der Gruppe real ist, da alle gleichwertig scheinen"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-In cryptography, a ring signature is a type of digital signature that can be performed by any member of a group of users that each have keys. Therefore, a message signed with a ring signature is endorsed by someone in a particular group of people. One of the security properties of a ring signature is that it should be computationally infeasible to determine *which* of the group members' keys was used to produce the signature.
+In der Kryptografie ist eine Ringsignatur eine Art digitaler Signatur, die durch ein beliebiges Mitglied einer Nutzergruppe, in der jeder die Schlüssel hat, erbracht werden kann. Das heißt, eine mit Ringsignatur signierte Nachricht wird von irgendjemandem aus einer bestimmten Gruppe gesendet. Eines der Sicherheitsmerkmale von Ringsignaturen ist, dass es rechnerisch unausführbar sein sollte zu bestimmen, *welches* der Gruppenmitglieder seine Schlüssel zum Erstellen der Signatur verwendet hat.
 
-For instance, a ring signature could be used to provide an anonymous signature from "a high-ranking White House official", without revealing which official signed the message. Ring signatures are right for this application because the anonymity of a ring signature cannot be revoked, and because the group for a ring signature can be improvised (requires no prior setup).
+Eine Ringsignatur könnte beispielsweise genutzt werden, um eine anonyme Signatur "eines hochrangigen Offiziellen des Weißen Hauses" bereitzustellen, ohne offenzulegen, welcher Offizielle die Nachricht signiert hat. Für diese Verwendung sind Ringsignaturen geeignet, da ihre Anonymität nicht aufgehoben werden und die Gruppe für eine Ringsignatur improvisiert werden kann (bzw. keine vorherige Einrichtung notwendig ist).
```

Comment:
```suggestion
Eine Ringsignatur könnte beispielsweise genutzt werden, um eine anonyme Signatur "eines hochrangigen Offiziellen des Weißen Hauses" bereitzustellen, ohne offenzulegen, welcher Offizielle die Nachricht signiert hat. Für diese Verwendung sind Ringsignaturen geeignet, da ihre Anonymität nicht aufgehoben werden und die Gruppe für eine Ringsignatur spontan (ohne vorherige Abstimmung) gebildet werden kann.
```

.. should also have someone check this technically.

Reply:
I think your suggestion doesn't have a problem with technical correctness, so I changed the text accordingly (I added "Einrichtung" though: "... spontan (ohne vorherige Abstimmung bzw. Einrichtung) gebildet werden kann").